docs/k0sctl-install.md

```diff
@@ -0,0 +1,171 @@
+
+## Deploying a k0s cluster using k0sctl
```

Comment:
Deployment chapter should explain the rough steps what k0sctl does in practice, like connect to the nodes using ssh, download k0s, install k0s as a service, connect nodes together securely.